src/metabase/lib_be/metadata/bootstrap.clj

```diff
@@ -0,0 +1,111 @@
+(ns metabase.lib-be.metadata.bootstrap
```

Comment:
I moved the "bootstrap" code in `qp.setup` responsible for resolving the database queries with `:database -1337` (a historic way to specify queries with a source card when the database ID of the source card was unknown) into `lib-be`, so `lib-be/normalize-query` can do this automatically.